metron-interface/metron-alerts/package-lock.json

```diff
@@ -5808,6 +5808,15 @@
       "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=",
       "dev": true
     },
+    "angular-confirmation-popover": {
+      "version": "4.2.0",
+      "resolved": "https://registry.npmjs.org/angular-confirmation-popover/-/angular-confirmation-popover-4.2.0.tgz",
+      "integrity": "sha512-ItCPzV52user93NRk9rF4Rp8NpawBWJdkNf8+6lH//f5i/N5HY0Aq5Hcch3xk19h9P48k0WZnfwOQL181xe4MQ==",
+      "requires": {
+        "positioning": "^1.3.1",
```

Comment:
@nickwallen According to semantic versioning and how npm handles these, The `^` sign at the beginning of the version means that any other "minor" updates of Positioning are accepted by the popover but not a major change.

1.4.0 is a minor update comparing to 1.3.1 since only the second number has been changed so npm installed and stored 1.4.0 which is the latest version and also the latest minor version of the package within version 1.